modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/service/impl/ClientExtensionEntryLocalServiceImpl.java

```diff
@@ -308,9 +308,6 @@ public ClientExtensionEntry updateClientExtensionEntry(
 			clientExtensionEntry.getTypeSettings(),
 			clientExtensionEntry.getType());
 
-		clientExtensionEntryLocalService.undeployClientExtensionEntry(
-			clientExtensionEntry);
-
```

Comment:
This makes sense, follows the pattern "first persist, then deploy" we added [here](https://github.com/liferay/liferay-portal/commit/70e133fde4765b6b0edb50442decdb510ecea8dc)